sunpy/net/dataretriever/sources/tests/test_norh.py

```diff
@@ -18,7 +21,45 @@ def LCClient():
     return norh.NoRHClient()
 
 
-@pytest.mark.remote_data
+def create_url(date, wavelength):
+    date = date.isoformat()
+    if wavelength == 34 * u.GHz:
+        freq = 'tcz'
+    elif wavelength == 17 * u.GHz:
+        freq = 'tca'
+    date = date.split('T')[0].split('-')
+    value_ = date[0][-2:] + date[1] + date[2]
```

Comment:
Would it be possible to do this with `.strftime()` - https://strftime.org/ is a good reference. If so, it would be easier to understand what's going on here (ie. which bits of the date/time are being extracted where)

Reply:
Yes, this makes more sense. I forgot this existed :sweat_smile: